app/assets/javascripts/actions/rooms_actions.es6.jsx

```diff
@@ -7,9 +7,12 @@
       );
     }
 
-    fetchRooms(page, conference_id) {
+    fetchRooms(conference_id, conferences) {
       var resolve = (response) => this.storeRooms(response);
-      Requester.get(ApiConstants.rooms.index(page, conference_id), resolve);
+      if (!conference_id) {
```

Comment:
Don't need this because will always have `conference` as determined by `rooms_controller` now.